consensus/istanbul/core/core.go

```diff
@@ -325,8 +325,14 @@ func (c *core) catchUpRound(view *istanbul.View) {
 	c.updateRoundState(view, c.currentCommittee, true)
 	c.roundChangeSet.Clear(view.Round)
 
+	newProposer, err := c.backend.GetProposerByRound(view.Sequence.Uint64(), view.Round.Uint64())
```

Comment:
Both (1) original `c.currentCommittee.Proposer()` and (2) patched `newProposer` are both purely for printing purpose. Original code always printed log. Let's keep that way. i.e. no early `return`.

The currentCommittee will be updated in startNewRound anyway.